### PR TITLE
[full-ci][tests-only]Run ocis tests without testing-app

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description

This PR removes the cloning of the testing app , as it's not really needed here

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -671,7 +671,6 @@ def localApiTests(ctx, storage, suite, accounts_hash_difficulty = 4):
                  restoreBuildArtifactCache(ctx, "ocis-binary-amd64", "ocis/bin") +
                  ocisServer(storage, accounts_hash_difficulty) +
                  restoreBuildArtifactCache(ctx, "testrunner", dirs["core"]) +
-                 restoreBuildArtifactCache(ctx, "testing_app", dirs["testing_app"]) +
```
Comment (phil-davis) [inline .drone.star:674]: Can we remove `SKELETON_DIR` at line 684?

File: .drone.star [hunk 2]
```diff
@@ -761,7 +760,6 @@ def coreApiTests(ctx, part_number = 1, number_of_parts = 1, storage = "ocis", ac
...
```
Comment (phil-davis) [inline .drone.star:764]: Can we remove `SKELETON_DIR` at line 773?

File: .drone.star [hunk 3]
```diff
@@ -879,7 +877,6 @@ def uiTestPipeline(ctx, filterTags, early_fail, runPart = 1, numberOfParts = 1,
                  ocisServer(storage, accounts_hash_difficulty) +
                  waitForSeleniumService() +
                  waitForMiddlewareService() +
-                 restoreBuildArtifactCache(ctx, "testing_app", dirs["testing_app"]) +
```
Comment (phil-davis) [inline .drone.star:882]: Can we remove `TESTING_DATA_DIR` at line 889?

Comment (phil-davis) [reply .drone.star:882]: Also, `cloneCoreRepos()` has `git clone` of `https://github.com/owncloud/testing.git`
Can that be removed?

**Comment by saw-jan** [comment]
NOTE: We need `testing` app for parallel deployment tests (but the pipelines are currently skipped)

CC @SwikritiT @phil-davis

**Comment by SwikritiT** [comment]
I've not removed the cloning of the testing app as it is need for parallel deployment

File: .drone.star [hunk 4]
```diff
@@ -147,8 +147,8 @@ config = {
     "dockerReleases": {
         "architectures": ["arm", "arm64", "amd64"],
     },
-    "litmus": True,
-    "codestyle": True,
+    "litmus": False,
+    "codestyle": False,
```
Comment (individual-it) [inline .drone.star:151]: why are we changing that?

Comment (SwikritiT) [reply .drone.star:151]: sorry I forgot to revert it

**Review by individual-it — APPROVED** [review]

**Pull request merged**